packages/components/src/Subscription/Component.purs

```diff
@@ -382,6 +386,14 @@ render self@{ props: props@{ subscription: sub@{ package } } } =
     billingPeriodEndDate =
           map trim $ formatDate =<< toMaybe props.subscription.dates.end
 
+    -- NOTE: We have a rule in our company policy that states that subscription pauses should be 7 days apart.
+    -- Thus, if a customer wants to extend a pause, they can't do it by adding a new pause immediately after it.
+    -- This is why we tell the customer to delete the pause and create a new one.
+    pauseDescription = DOM.div
+                         { className: "mitt-konto--note"
+                         , children: [ DOM.text "Om du vill ändra på din tillfälliga adressändring eller ditt uppehåll, vänligen radera den gamla först och lägg sedan in den nya." ]
+                         }
```

Comment:
This is just temporary before we implement backend magic to that users don't have to care about this, right?
Or is this meant to be permanent?

Reply:
We have no idea at the moment, first we have to understand our company policy better 😄